docs_src/manuals/getting_started/CMakeLists.txt

```diff
@@ -54,7 +54,6 @@ ExternalProject_Add(getting_started_cmake_test
     DOWNLOAD_COMMAND ""
     SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/code
     CMAKE_ARGS
-        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}  # Necessary for HPX to work
```

Comment:
Let's keep this since we check that the `CMAKE_BUILD_TYPE` is set and is the same as the one used to build hpx in the hpx configuration step (this will fail otherwise), sorry if that's annoying to add :/

Reply:
Just saw that is already solved !

Reply:
It doesn't fail for me. Let's wait for CI.

Reply:
Ah now I understand why: in your version there was `add_dependency(HPX)` in the GridToolsConfig.cmake. But that's not necessary as we do this in setup_cmake_targets.cmake. Now when we build the getting_started hpx will be silently not found (which is maybe not perfect, but ok for now).

Reply:
Yep indeed!